versions.json

```diff
@@ -8,7 +8,7 @@
     "tgt": {
         "repo": "https://github.com/rancher/tgt.git",
         "commit": "3a8bc4823b5390e046f7aa8231ed262c0365c42c",
-        "tag": "v1.0.79+3a8bc48",
+        "tag": "v1.0.79.1",
```

Comment:
Is this a semantic version? Or it should be 1.079-1?

Reply:
Good question...
AppCo doesn't support sematic version as discussed in https://suse.slack.com/archives/C08DAJ827NX/p1741968925723489?thread_ts=1741831970.776719&cid=C08DAJ827NX.

Reply:
After discussing with @innobead, we will use v<x.y.z>+<revision>.